src/utils/cellar.ts

```diff
@@ -0,0 +1,236 @@
+import {
+  BigDecimal,
+  BigInt,
+  log,
+} from '@graphprotocol/graph-ts'
+
+import {
+  CellarContract,
+  CellarContract__cellarTickInfoResult,
+} from '../types/Cellar/CellarContract'
+import {
+  Cellar,
+  NFLP,
+  Pool,
+  Token,
+} from '../types/schema'
+import {
+  ONE_BD,
+  ONE_BI,
+  ZERO_BD,
+  ZERO_BI,
+} from './constants'
+
+export function loadCellar(cellarAddress: string): Cellar {
+  log.debug('Invoked loadCellar, address: {}', [cellarAddress])
+  let cellar = Cellar.load(cellarAddress)
+  if (cellar == null) {
+    throw new Error('Could not find Cellar:'.concat(cellarAddress))
+  }
+
+  return cellar as Cellar
+}
+
+export function initCellar(contract: CellarContract, cellarAddress: string): Cellar {
+  log.debug('Invoked initCellar', [])
+  let cellar = new Cellar(cellarAddress)
+  cellar.token0 = contract.token0().toHexString()
+  cellar.token1 = contract.token1().toHexString()
+
+  // TODO: derive this from token and fee
+  // PORT: https://github.com/Uniswap/uniswap-v3-sdk/blob/main/src/utils/computePoolAddress.ts
+  cellar.pool = '0x8ad599c3a0ff1de082011efddc58f1908eb6e6d8' // USDC-ETH 0.3%
```

Comment:
we'll need to figure out how to port computePoolAddress so we can keep this generic. I think we can launch with hardcoded pool addresses and won't need to port until the cellar factory is live.